isyfact-standards-doc/src/docs/antora/modules/changelog/pages/migrationsleitfaden.adoc

```diff
@@ -17,10 +17,238 @@ Die auf dieser Seite aufgeführten Hinweise sollen IsyFact Anwendungsentwicklern
 [[kapitel-batchrahmen]]
 === Batchrahmen
 
-
 [[kapitel-sicherheit]]
 === Sicherheit
 
+Die Bausteine `isy-sicherheit` und `isy-sicherheit-keycloak` werden durch `isy-security` abgelöst.
+
+[[sicherheit-konfiguration]]
+==== Konfiguration
+
+Eine Beschreibung der Parameter findet sich in den xref:isy-security:nutzungsvorgaben/master.adoc#konfigurationsparameter[Nutzungsvorgaben Security] und der xref:glossary:literaturextern:inhalt.adoc#litextern-spring-security-properties[Dokumentation von Spring Security].
+
+<<table-sicherheit-konfiguration>> gibt eine Übersicht über die zu ändernden Parameter.
+Die `instances[n]` Notation aus den isy-sicherheit-keycloak Properties wird in isy-security nicht länger unterstützt.
+Jede Instanz soll durch einen sprechenden und eindeutigen `provider-name` bzw. eine `registration-id` ersetzt werden.
+
+Aus Platzgründen wurde in der ersten Spalte auf das `isy.sicherheit.keycloak` Präfix verzichtet.
+Analog wurde bei allen Properties der zweiten Spalte, die mit einem Punkt beginnen, auf das Präfix `spring.security.oauth2.client` verzichtet.
+
+:table-sicherheit-konfiguration: Änderung der Konfigurationsparameter
+
+[id="table-sicherheit-konfiguration",reftext = "{table-caption} {counter:tables}"]
+.{table-sicherheit-konfiguration}
+[cols="1,1,1"]
+|===
+|isy-sicherheit(-keycloak)|isy-security|Bemerkung
+|`.auth-server-url`|`.provider.<provider-name>.issuer-uri`| für die Verwendung in Kombination mit einer Spring Client Registration, an den bisherigen Wert wird `/realms/<realm-name>` angehängt
+|`.auth-server-url`| `spring.security.resourceserver.jwt.issuer-uri`| für die Verwendung in Kombination mit _@Secured_, an den bisherigen Wert wird `/realms/<realm-name>` angehängt
+|`.realm`|-|entfällt, da Teil der `issuer-uri`
+|-|`.registration.<registration-id>.provider`|neu, verbindet die Spring Client Registration und Provider Definition
+|`.resource`|`.registration.<registration-id>.client-id`|
+|`.credentials-secret`|`.registration.<registration-id>.client-secret`|
+|-|`.registration.<registration-id>.authorization-grant-type=password`| neu, da ein zusätzlicher OAuth Flow unterstützt wird
+|`.bearer-only`|-|entfällt
+|`.standard-zertifikat-ou`|`isy.security.oauth2.client.default-certificate-ou`|
+|-|`isy.security.oauth2.registration.<registration-id>.username`|neu, wurde vorher aus dem AufrufKontext ausgelesen
```

Comment:
The properties `username` and `password` are only defined for **Confidential Clients** independent on _isy-sicherheit_ or _isy-security_. But with _isy-security_ the configuration of a _spring conform ClientRegistration_ is introduced, which expects the _authentication properties_ on a dedicated path. This means the previous defined properties `username` and `password` of _batch-_ or _task_-configurations, have to be shifted to the new property-location. This mechanism will only be used by the `Security`-Interfacemethod `authentifizieren`.

All other `Security`-Interfacemethods will get these values as parameters through external application invocations. In these cases _isy-security_ will transfer the _authentification data_ in the internal `SecurityContext` analog to the `AufrufKontext` in _isy-sicherheit_.

```suggestion
|-|`isy.security.oauth2.registration.<registration-id>.username`| für _ConfidentialClient_-Konfiguration der geänderte Pfad für Property `username`, mit _isy-security_ nun innerhalb einer _Spring konformen ClientRegistration_
```

Reply:
please see new suggestion below